### ISSUE TITLE
Upload interaction design

### ISSUE THREAD

**Issue by shiffman** [ml5js/ml5-homepage-demo#3]
I'll defer to @matamalaortiz on the final design here but I think the language for uploading a file is a bit wordy and it's confusing where the "here" is to drag the image. I think a basic "upload image" button might work better as well as some visual indicator that you can drag drop onto the image?

**Comment by yining1023** [comment]
I agree. Maybe use an emoji to hint dropping file? We are using emojis at other places on the website.
I'm just brainstorming. How about this? @matamalaortiz @shiffman 
<img width="400" alt="screen shot 2018-06-06 at 11 43 31 pm" src="https://user-images.githubusercontent.com/8662372/41077304-ee9bd7c0-69e3-11e8-8f59-9ba88de52ab2.png">

**Comment by matamalaortiz** [comment]
I agree too. 
What about the Upload Image button in the center and beneath and a little smaller or drag and drop and the emoji. ?

**Comment by shiffman** [comment]
Sounds good, I agree the upload info should be right below the image. I think the text can also be combined into one sentence (sticking with multi-color but avoiding black?)

**Comment by yining1023** [comment]
I made some style updates. It looks like this now. I might have missed something. Please feel free to change it.
<img width="800" alt="screen shot 2018-06-07 at 2 07 25 pm" src="https://user-images.githubusercontent.com/8662372/41117875-40e4be48-6a5c-11e8-92c7-ff579b0cc203.png">

**Comment by cvalenzuela** [comment]
Looks great! I really like it
